.buildkite/scripts/set-bk-analytics-token.ps1

```diff
@@ -0,0 +1,3 @@
+Write-Host "--- Prepare BK test analytics token :vault:"
+$BUILDKITE_ANALYTICS_TOKEN = & vault kv get -field=token kv/ci-shared/platform-ingest/buildkite_analytics_token
```

Comment:
similar to the other comment, should we define some flags that would prevent execution if there is a failure

Reply:
and potentially add a retry?

Reply:
I was wrong. Retrying is a bit more complicated, so it might make sense to follow up and add it later.

Reply:
Added retries